consul_config.py.ctmpl

```diff
@@ -26,7 +26,12 @@ MUSICBRAINZ_CLIENT_SECRET = '''{{template "KEY" "musicbrainz/client_secret"}}'''
 {{if service "pgbouncer-master"}}
 {{with index (service "pgbouncer-master") 0}}
 SQLALCHEMY_DATABASE_URI = "postgresql://critiquebrainz:critiquebrainz@{{.Address}}:{{.Port}}/critiquebrainz_db"
-MB_DATABASE_URI = "postgresql://musicbrainz_ro@{{.Address}}:{{.Port}}/musicbrainz_db"
+{{end}}
+{{end}}
+
+{{if service "postgres-slave"}}
```

Comment:
This should use the pooler, via `pgbouncer-slave`. There's only a limited number of direct connections available.

Reply:
Okay, whoops. I'll fix and deploy today.